googledatastore/src/main/java/com/yahoo/ycsb/db/GoogleDatastoreClient.java

```diff
@@ -243,7 +242,6 @@ public Status scan(String table, String startkey, int recordcount,
   @Override
   public Status update(String table, String key,
       HashMap<String, ByteIterator> values) {
-
     return doSingleItemMutation(table, key, values, MutationType.UPDATE);
```

Comment:
One empty line is missing here in the new code. 

There has been a reformatting effort recently to all the code, please make sure this PR retains the outcome of that reformatting effort.